src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -757,14 +757,15 @@ class ConsignmentRouteSpec extends TestContainerUtils with Matchers with TestReq
     response.errors.head.extensions.get.code should equal("NOT_AUTHORISED")
   }
 
-  "startUpload" should "add the upload status and update the parent folder" in withContainers { case container: PostgreSQLContainer =>
+  "startUpload" should "add the upload status and update the parent folder and 'IncludeTopLevelFolder' fields" in withContainers { case container: PostgreSQLContainer =>
```

Comment:
Maybe reword test slightly to: `... upload status, update the parent folder and ...`